src/lib/SpotlightTour.context.ts

```diff
@@ -43,6 +43,7 @@ export const SpotlightTourContext = createContext<SpotlightTourCtx>({
   goTo: () => undefined,
   next: () => undefined,
   previous: () => undefined,
+  spot: { height: 0, width: 0, x: 0, y: 0 },
```

Comment:
I see this value in other places too, should we export our `ZERO_SPOT` constant and use that instead?

Reply:
The React context initialization is more of a "type-safety" requirement. But it's not a bad idea to export the `ZERO_SPOT` constant from here to keep things DRYer. Thanks 🙂